utils.py

```diff
@@ -44,14 +47,32 @@ def create_font(font, size):
         print("Font not available")
 
 
-def get_btc_price(fiat_code):
-    """Get BTC -> FIAT conversion
+def get_btc_price():
+    """Get BTC Price in usd(t)
     """
-    url = config.COINGECKO_URL_BASE + "simple/price"
```

Comment:
The "old" solution should be switchable in the config.
Many users want to use their own local currency